src/screens/ContactScreen/index.js

```diff
@@ -264,7 +264,7 @@ const styles = StyleSheet.create({
     marginHorizontal: 16
   },
   subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.holyTosca : COLORS.gray4,
+    color: selectedUsers.length > 0 ? COLORS.holytosca : COLORS.gray4,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant `COLORS.holytosca` has been changed to lowercase. Please ensure that the color constants are correctly defined in the `COLORS` object and used consistently throughout the codebase.
```diff
-    color: selectedUsers.length > 0 ? COLORS.holyTosca : COLORS.gray4,
+    color: selectedUsers.length > 0 ? COLORS.holytosca : COLORS.gray4,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->